game/nintendo_mii/gen1_wii_mii.ksy

```diff
@@ -0,0 +1,204 @@
+meta:
+  id: gen1_wii_mii
+  file-extension:
+    - mii
+    - mae
+    - miigx
+    - rcd
+    - rsd
+  endian: be
+seq:
+  - id: invalid
+    type: b1
+    doc: Has no apparent effect.
+  - id: gender
+    type: b1
+    doc: Mii gender. 0 = male, 1 = female.
```

Comment:
How about `is_female`?

Reply:
Leaving the name `gender` and connecting the attribute with the enum `{ 0: male, 1: female }` is better.

> How about `is_female`?

The catch with `is_female` is that it doesn't really say what does it mean when the value is `false` - the only thing that you know for sure is that it _isn't female_, and it actually might be anything. Explicit is better than implicit.

Reply:
I agree. I decided to just keep it as it was before I started working on it, for the time being, but I'm glad you agree.

Reply:
> Keeping the name gender and connecting the attribute with the enum { 0: male, 1: female } is better.

This also seems better, but I was unsure if it was necessary for a format that unfortunately considers gender to be binary.

Reply:
Can this be resolved now? Is the current solution agreeable?